lambda/es-proxy-layer/lib/handler.js

```diff
@@ -68,15 +68,19 @@ async function get_settings() {
     return settings;
 }
 
-
 async function get_es_query(event, settings) {
-    var question = _.get(event,'question','');
+    let question = _.get(event,'question','');
+    let size = _.get(event,'size',1);
     if (question.length > 0) {
+        if (open_es.isQuestionAllStopwords(question)) {
+            console.log(`Question '${question}' contains only stop words. Forcing no hits.`);
```

Comment:
Could you please provide steps to tests this fix (test case with required assets (e.g., sample inputs)), so our QA can added to our test cases for the coming release.

Reply:
Sure.. To test try input message to bot with only stopwords.. eg "and not a"  - before this fix it would return some random answer, with this fix it should return no_hits ("sorry - your stumped me". try it in both ContentDesigner TEST tab, and in the bot web client.